src/shared/components/simple-select/SimpleSelect.test.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useRef, useEffect } from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import SimpleSelect, { type SimpleSelectMethods } from './SimpleSelect';
+
+const options = [
+  {
+    value: '1',
+    label: 'One'
+  },
+  {
+    value: '2',
+    label: 'Two'
+  },
+  {
+    value: '3',
+    label: 'Three'
+  }
+];
+
+describe('SimpleSelect', () => {
+  it('renders a list of options', () => {
+    const { container } = render(<SimpleSelect options={options} />);
+    const simpleSelectElement = container.firstElementChild as HTMLElement;
+
+    // SimpleSelect renders a div element (for styling purposes)
+    expect(simpleSelectElement.tagName.toLowerCase()).toBe('div');
```

Comment:
Wouldn't checking if the `select` class exists be more useful here?

Reply:
Added checks for the `select` class and for a CSS class passed from the parent. I think there's still value in checking that the top-level element rendered by this component is not a `select`.